modules/administration-guide/partials/assembly_backups-of-che-instances-to-a-rest-backup-server.adoc

```diff
@@ -0,0 +1,11 @@
+[id="backing-up-{prod-id-short}-instances-to-a-rest-backup-server_{context}"]
+= Backups of {prod-short} instances to a REST backup server
```

Comment:
Should the title of this assembly, and others like it, begin with a gerund instead of a noun? For example:
```suggestion
= Backing up of {prod-short} instances to a REST backup server
```

Reply:
@rolfedh, yes, it should, thank you. I just didn't think of `<gerund> of` at the time.